Lib/test/test_asyncio/test_tasks.py

```diff
@@ -1147,22 +1147,6 @@ def gen():
         res = loop.run_until_complete(task)
         self.assertEqual(res, "ok")
 
-    def test_wait_for_cancellation_race_condition(self):
```

Comment:
Does this test now fail if it wasn't removed?

Reply:
Exactly. The test is incorrect (the task _must_ raise) and it doesn't reproduce original problem.